src/caret_analyze/plot/histogram/histogram_factory.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2021 Research Institute of Systems Planning, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+from typing import Sequence
```

Comment:
If `annotations` are to be introduced, I think `typing` is not necessary.

Reply:
Fixed in https://github.com/tier4/CARET_analyze/pull/289/commits/14681c2f0479dce989b519d04afd33530227d5d7